### ISSUE TITLE
Only logout client if really inactive (currently also logs him out if he's just offline)

### ISSUE THREAD

**Issue by SimonSimCity** [lindleycb/meteor-stale-session#20]
Could you please help me wrapping my head around the following scenarios?

1. A user is using my meteor-application for some time, switching his device off for a while. He should be asked to log in again.
2. A user is using my meteor-application and looses the network connection for a while. He's still actively working with the data on the screen. Once he connects back, he should remain logged in.

Is there a possibility where I can inform the server that the client has been active in the meanwhile, before he kicks him out? I thought about calling `Meteor.logout()` if `this.userId` is set, but `activityDetected` is not, in https://github.com/lindleycb/meteor-stale-session/blob/master/client.js#L20. Do you think this would be a good approach? Another option would be if you call the `heartbeat` method with a timestamp. Then you could ensure that the next timestamp needs to be `heartbeatInterval` away from the last (with some margin). Since the system now saves the requests if the connection is down and tries to resend as soon as it's available, this could work ... what do you think?

**Comment by lindleycb** [comment]
Sorry to have been so lax in the maintenance of this package but congratulations on getting a package together that solves your problem.

**Comment by SimonSimCity** [comment]
@lindleycb Now facing back, would you merge them together or leave them separate? To me it's a different approach to solve a slightly different task, because your plugin needs to send the heartbeat. It could be that the server already purged your session out 😉  Means, I'd have to go online within the `staleSessionInactivityTimeout`, right?

**Comment by lindleycb** [comment]
Yes I agree.  The concerns are slightly different so I'd say take a unix-like approach and have multiple small packages that can be used together or apart as developers see fit.
